src/harness/fake_sas.py

```diff
@@ -35,6 +65,8 @@
     'AES128-GCM-SHA256', 'AES256-GCM-SHA384', 'ECDHE-RSA-AES128-GCM-SHA256'
 ]
 
+MISSING_PARAM = 102
+INVALID_PARAM = 103
```

Comment:
FYI in other files, we've been using the number directly rather than defining a constant. I think either is fine, though (no change needed).